packages/app/providers/auth-provider.tsx

```diff
@@ -137,10 +155,34 @@ export function AuthProvider({
 
   //#region effects
   useEffect(() => {
-    if (authenticationStatus === "IDLE") {
+    if (!initialRefreshTokenRequestSent.current) {
       doRefreshToken();
```

Comment:
here we don't keep 6-hour check so not storing it in a persistent storage. Thought it would be early optimization.